src/main.ts

```diff
@@ -35,7 +35,9 @@ async function start() {
 	// We are always behind a proxy, but we want the source IP
 	probot.server.set("trust proxy", true);
 	probot.load(App);
-	probot.webhook.on("error", (err: Error) => logger.error({ ...err }, "Webhook Error"));
+	probot.webhook.on("error", (err: Error) => {
+		logger.error({...err, err: JSON.stringify(err)}, "Webhook Error")
```

Comment:
Not sure what magic happens in Probot, but it adds an event to the exception. If we log it like {...err} we'll see an event only and loose the stack trace. If we do JSON.stringify(err) we'll get an error message. 

So added it to have both for now

Reply:
it's not probot, it's bunyan.  It only picks a few properties out of Error to display.